README.md

```diff
@@ -320,7 +320,7 @@ The configuration generated with this module is only compatible with newer relea
 Please fork this repository, hack away on your branch, run the tests:
 
 ```shell
-$ bundle exec rake test acceptance
+$ bundle exec rake test spec/acceptance
```

Comment:
```
bundle exec rake beaker
```
is probably the better way to handle this. Also our .github/CONTRIBUTUNG.md explains how to run the acceptance tests on different operating systems.

Reply:
Thank you for reviewing.
Then, this command was wrong in any case, so I fixed it.